src/services/item/plugins/importExport/schema.ts

```diff
@@ -25,18 +25,34 @@ export const zipImport = {
 } as const satisfies FastifySchema;
 
 export const zipExport = {
+  operationId: 'exportZip',
+  tags: ['item', 'export'],
+  summary: 'Export non-folder content',
+  description: 'Export non-folder content. Return raw file for single item.',
+
```

Comment:
Should this still be called "zip" ? I can not see when this would be a zip if we can't export folders. It should always be a single file (image, text, html). maybe for h5p it could be a `.h5p` which is a zip inside ... but that is all I think.

I would call this "download Item" to make it evident that it gives the data directly. 
"Export" has a sort of "take it out of the platform + post-processing to it I think.